frontend/app/actions/persistZip.jsx

```diff
@@ -0,0 +1,5 @@
+export const PERSIST_ZIP = 'PERSIST_ZIP'
```

Comment:
This needs to clear if the user navigates back to the main page and searches a different zip. At the moment it just persists the first zip you typed for the entire session.